### DISCUSSION TITLE
How to edit the  "No Notificaitons" message and/or icon?

### DISCUSSION THREAD

**Discussion by setseed13** [ErikReider/SwayNotificationCenter#483] [Q&A]
Hi all!

I keep seeing themes which have a different icon shown, when there are no notifications. However, even after digging through the respective configs, I can't find a way to replace the stock icon and/or the message displayed.

Is this hardcoded and changed over time, or am I just looking in the wrong places?

Thanks@

**Comment by ffs-akl** [comment]
that probably is hardcode.

**Comment by yerlotic** [comment]
Since version v0.11.0 you can use `"text-empty"` in your `config.json`

```jsonc
{
    // stuff

    "text-empty": "My custom text",
    
    // other stuff
}
```

However, there is no way to change the icon dynamically (yet)

**Comment by setseed13** [comment]
thanks, at least a start :)
